tests/braille-specs/hu-hu-g1_braille_input_backward.yaml

```diff
@@ -177,3 +177,17 @@ tests:
   - [⠼⠁⠜⠜, "1zzs"]
   - [⠼⠁⠜⠜⠼⠁, "1zzs1"]
   - [⠼⠁⠠⠁⠣⠣⠎, "1azzs"]
+#Following testcases testing redesigned old compatibility purpose translation rules,
+#with affects only back-translation
```

Comment:
I don't understand the comments.

Reply:
Bert, need this change doing because the compatibility purpose rules produces wrong working method with forward translation if a text contains for example with unicode Braille characters.
So, both forward and backward translation tests we uses equals Braille dot combinations, but wait different working method now.
Look few example for example with tests/braille-specs/hu-hu-g1_braille_input_forward.yaml:
```
  - [⠐⠷, ⠐⠷]
  - [⠐⠾, ⠐⠾]
```
So, in forward direction you need see this dot combination the braille display if the text contains unicode braille characters (the braille display need presents with 5-12356 and 5-23456 dot combination).

Looks what need happens if the user typewriting this dots he's braille display, this test case shows the hu-hu-g1_braille_input_backward.yaml test file:
```
  - [⠐⠷, "["]
  - [⠐⠾, "]"]
```

So, the real testcase examples easyer understandable. The 5-12356 dot combination compatibility purpose rule need execute only with back-translation only, and need results a [ character. But if happens forward translation, the 5-12356 unicode braille output characters the text editor need presents real phisical Braille display with 5-12356 dot combination too.
I don't no how possible writing right grammar documentation this testcase elements related, but human tests confirm both forward and backward translation everithing is works right now and this bug is resolved with old compatibility purpose rules with Braille inputs related if an user want use old compatibility dot combinations the punctuation characters related.
This old dot combinations now not disturb forward translation output if have a pasted unicode Braille character the text file, for example an unicode 5-12356 braille output character sequence. Previous table versions the unicode braille 5-12356 dot combination presents with 46-2346 dot combination the braille displays, because we use noback pass2 @5-12356 @46-2346 dot combination. We want this if an user typing for example the 5-12356 dot combination, but not know happens too this replacement if  an unicode Braille text contains for example with 5-12356 dot combination).
So, we drop the noback direction compatibility purpose rules, and keeps only the nofor direction compatibility rules.

Bert, feel free you drop the comments if not have better idea the documentation entries related (me not have right documentation ydea now).

Attila

Reply:
This is no use 😞. How can I help with grammar and documentation if I can't understand a word of what you're saying? I'm sorry Attila. Sometimes I can kind of understand you but sometimes it's too hard. It really frustrates me (and I'm sure it's as frustrating for you).

Reply:
Absolute. :-):-)
Some time easyest i write a goood understandable answer, but some time not.
This rule change working method too complex to I send a good documentation comment.
I tried translate  Google Translator my remaining comment part from hungarian language to english language.
So, the point is, if there are 5-12356 unicode braille characters have in a text, then a combination of 5-12356 points must appear on the Braille display.
But:
If the user types the dot combination 5-12356 on the braille display, the normal text editor should display a ] character when translated with normal text (this situation happen simple bakc-translation, because we use nofor direction).
With the old 3.26.0 table version because we used the noback pass2 @5-12356 @46-2346 dot combination, the Unicode Braille character with the 5-12356 dot combination was incorrectly displayed as the 46-2346 dot combination, not the original 5-12356 dot combination.
I fixed this now.

Hopefully Google Translator translate right I want say this rules related in hungarian language to english language, and this comment are easyer understand. :-):-)
Attila

Reply:
Boys,, we what doing into the  test files related comment parts both tests/braille-specs/hu-hu-g1_braille_input_forwards.yaml and tests/braille-specs/hu-hu-g1_braille_input_backwards.yaml tests?
Because release date very near, simple drop out the unneed comment parts from testcase files?

Attila

Reply:
Hi Attila, we delayed the release one week. There should be time enough now.

Thanks to Google Translator I now finally get what this is about 😃. I suggest we just drop the comments in the test file, because basically these new tests are not different from the existing ones. We explain the improvement in the NEWS file.

Reply:
Oops, you are right... :-):-)
Release date is real september 12, but I thinked you will be a well-deserved vacation between august 28 beginning week and next week. :-):-)
I misunderstant Chris following letter part from the Liblouis sent notification:
> Turns out Bert will be on some vacation next week and we
> will not have any time to prepare the release.
>
> So we will have to delay the release.
>
> The new release date is Tuesday, September 12th
>
>Bert and I will work on integrating the Pull Requests in the week of
September 4th. So make sure your contributions will be ready by then.

Final I removed the unneed comment parts both two direction test case files before the new test elements.

Attila